src/cascadia/TerminalApp/defaults.json

```
@@ -340,6 +340,17 @@
         // Visual Adjustments
         { "command": { "action": "adjustFontSize", "delta": 1 }, "keys": "ctrl+=" },
         { "command": { "action": "adjustFontSize", "delta": -1 }, "keys": "ctrl+-" },
-        { "command": "resetFontSize", "keys": "ctrl+0" }
+        { "command": "resetFontSize", "keys": "ctrl+0" },
+
+        {
+            "name": { "key": "SetColorSchemeParentCommandName"},

```

wow is this our first keyed command? neat!

```suggestion
            "name": { "key": "SetColorSchemeParentCommandName" },
```